vendor/vendor_updater.ml

```diff
@@ -0,0 +1,37 @@
+open Import
+
+(* Initialize Path module and console backend *)
+let () =
+  (* Find project root by looking for dune-project *)
+  let rec find_up dir =
```

Comment:
Why can't we use our existing root finding?

Reply:
It's copy-pasted since I didn't want to move it out of `bin`. I can put it in `dune_util` however for reuse.

Reply:
How about just stdune?